repositories/go_repositories.bzl

```diff
@@ -37,7 +37,7 @@ def go_deps(go_repository_default_config = "@//:WORKSPACE"):
         go_repository_default_config (str, optional): A file used to determine the root of the workspace.
     """
     go_rules_dependencies()
-    go_register_toolchains()
+    go_register_toolchains(version = "1.18")
```

Comment:
Oh, this is weird. I don't know what the behaviour of calling `go_register_toolchains` twice is, especially with an explicit version.

Maybe this should change? I'm afraid of how it may conflict with an existing SDK.